service/history/hsm/registry.go

```diff
@@ -59,7 +59,6 @@ type Registry struct {
 	// The executor maps are mapped to any because of Go's limited generics support.
 	// The actual value is ImmediateExecutor[T].
 	activeImmediateExecutors  map[int32]any
-	standbyImmediateExecutors map[int32]any
 	// The actual value is TimerExecutor[T].
 	activeTimerExecutors  map[int32]any
 	standbyTimerExecutors map[int32]any
```

Comment:
You don't need these either now.

Reply:
Can address in the PR for timer standby executors.

Reply:
Gonna do in a separate PR.